friends/src/components/Friends.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import './friends.css';
+import Edit from '../img/edit.png'
+
+const Friends = (props) => {
+    return (
+        <div className="friend">
+            <div className="editIcon">
+                <img className="edit" src={Edit} onClick={ () => props.edit(props.friends.id)}/>
+            </div>
+            <div className="id">{props.friends.id}</div>
+            <div className="name"> {props.editing && props.friends.id == props.number ?
```

Comment:
awesome job setting up an editing state as a toggle to render each update form on click